submissions/adisen/Week 1/Fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,19 @@
+function fizzBuzz(){
+  for (var i = 1; i <= 100; i++) {
+    if (i % 3 == 0 && i % 5 == 0) {
+      console.log("FizzBuzz");
+    }
+    else if (i % 3 == 0) {
+      console.log("Fuzz");
+      continue;
```

Comment:
There is no reason to have a `continue` statement in each else block. There is no code that needs to be skipped after the `if else`. Also the `if else ` statement guarantees only one of the blocks will execute based on the value of `i`. 
You can remove the `continue` statement as it is unnecessary here.